documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -40,6 +40,30 @@ spec:
     # ...
 ----
 
+[[cors-configuration]]
+== Cross-origin resource sharing configuration
```

Comment:
I would use capital letters for making CORS ... "Cross-Origin Resource Sharing configuration"